templates.dist/fields.php

```diff
@@ -172,14 +175,16 @@ function renderFormField(string $fieldName, onOffice\WPlugin\Form $pForm, bool $
 				}
 				$htmlOptions .= '<option value="' . esc_attr($key) . '".' . ($isSelected ? ' selected' : '') . '>' . esc_html($value) . '</option>';
 			}
-			$output = '<select class="custom-multiple-select form-control" name="' . esc_html($fieldName) . '[]" multiple="multiple" ' . $requiredAttribute . '>';
+			$output = '<select name="' . esc_html($fieldName) . '[]" multiple="multiple" ' . $requiredAttribute;
+			$output .= $isHiddenField ? 'class="oo-hidden-field" disabled ' : 'class="custom-multiple-select form-control"' . '>';
 			$output .= $htmlOptions;
 			$output .= '</select>';
 		} else {
 			$inputType = 'type="text" ';
 			$value = 'value="' . esc_attr($pForm->getFieldValue($fieldName, true)) . '"';
-
-			if ($typeCurrentInput == onOffice\WPlugin\Types\FieldTypes::FIELD_TYPE_BOOLEAN) {
+			if ($isHiddenField === true) {
+				$inputType .= 'class="oo-hidden-field" disabled ';
```

Comment:
Please remove:
```
if ($isHiddenField === true) {
	$inputType .= 'class="oo-hidden-field" disabled ';
} else
```

to restore the original if statement:
```
if ($typeCurrentInput == onOffice\WPlugin\Types\FieldTypes::FIELD_TYPE_BOOLEAN) {
```

afterwards to make it work please insert the following `$inputType .= $hiddenAttribute;` around line 210 

```
} elseif (
	$typeCurrentInput === FieldTypes::FIELD_TYPE_DATETIME
) {
	$inputType = 'type="datetime-local" step="1" ';
}

// here

if (
	$isRangeValue && $pForm->inRangeSearchcriteriaInfos($fieldName) &&
	count($pForm->getSearchcriteriaRangeInfosForField($fieldName)) > 0
) {
```

Bug:
Currently when a checkbox is set to hidden the input type="" is stuck on "text" when it should be "checkbox". This causes an incorrect value in the field and they aren't submitted in the form. 

<img width="1149" alt="hidden" src="https://github.com/user-attachments/assets/82952c8d-66ad-4b5e-a7c0-181272703647">